docs/Tutorials/Create-Private-Clique-Network.md

```diff
@@ -0,0 +1,271 @@
+```
+
+### 6. Start Node-2 
+
+You need the enode URL for Node-1 to specify Node-1 as a bootnode. 
+
+The enode URL is `enode://<id>@<host:port>` where:
+
+* `<id>` is the node public key excluding the initial 0x. The node public key for Node-1 was written to `publicKeyNode1` in [step 2](#2-get-public-key-for-node-1). 
+* `<host:port>` is the host and port the bootnode is listening on for P2P peer discovery. Node-1 is using the default host and port of `127.0.0.1:30303`. 
+
+!!! example
+    If the default host and port are used for P2P peer discovery and the node public key exported is: `0xc35c3ec90a8a51fd5703594c6303382f3ae6b2ecb9589bab2c04b3794f2bc3fc2631dabb0c08af795787a6c004d8f532230ae6e9925cbbefb0b28b79295d615f`
+    
+    The enode URL is:
+    `enode://c35c3ec90a8a51fd5703594c6303382f3ae6b2ecb9589bab2c04b3794f2bc3fc2631dabb0c08af795787a6c004d8f532230ae6e9925cbbefb0b28b79295d615f@127.0.0.1:30303` 
+
+Start another terminal, change to the `Node-2` directory and start Node-2 specifying:
+ 
+* Different port to Node-1 for P2P peer discovery using the [`--p2p-port`](../Reference/Pantheon-CLI-Syntax.md#p2p-port) option.
+* Different port to Node-1 for HTTP JSON-RPC using the [`--rpc-http-port`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-port) option.
+* Enode URL for Node-1 using the [`--bootnodes`](../Reference/Pantheon-CLI-Syntax.md#bootnodes) option.
+* Data directory for Node-2 using the [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) option.
+* Other options as for [Node-1](#5-start-first-node-as-bootnode).  
+
+```bash tab="MacOS"
+pantheon --data-path=Node-2-data-path --genesis-file=../cliqueGenesis.json --bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --network-id 123 --p2p-port=30304 --rpc-http-enabled --rpc-http-api=ETH,NET,CLIQUE --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8546     
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-2-data-path --genesis-file=..\cliqueGenesis.json --bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --network-id 123 --p2p-port=30304 --rpc-http-enabled --rpc-http-api=ETH,NET,CLIQUE --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8546     
+```
+
+### 7. Start Node-3
+
+Start another terminal, change to the `Node-3` directory and start Node-3 specifying: 
+ 
+ * Different port to Node-1 and Node-2 for P2P peer discovery using the [`--p2p-port`](../Reference/Pantheon-CLI-Syntax.md#p2p-port) option.
+ * Different port to Node-1 and Node-2 for HTTP JSON-RPC using the [`--rpc-http-port`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-port) option.
+ * Data directory for Node-3 using the [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) option.
+ * Bootnode as for [Node-2](#6-start-node-2).
+ * Other options as for [Node-1](#5-start-first-node-as-bootnode). 
```

Comment:
Other options as for Node-1 and Node-2 ?

This confused me because I copied the command line text to start Node-2 (without reading the bullet points) and then when this said "as for Node-1" I had to go back and look at Node-1, not realising that Node-2 also said "as for Node-1"...

Reply:
Updated to move the bullet points below the command line given to make it clearer that it should be copied.